app/templates/components/question-single.hbs

```diff
@@ -14,6 +17,9 @@
 {{/if}}
 
 <h3 class="moduleHeader">{{question.title}}</h3>
+<div class="question-description">
+    {{{question.description}}}
```

Comment:
Since this property is special in that it can accept HTML (or what exactly can it display when using the triple braces?), please make a comment on the `question` model describing what is allowed in that field.